src/libstd/primitive_docs.rs

```diff
@@ -255,6 +255,11 @@ mod prim_unit { }
 ///
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
+/// [`null`]: ../std/ptr/fn.null.html
```

Comment:
Do you recall what those issues are? I know there are issues regarding linking things from libcollections because things are reexported, but wasn't aware of issues with core or std.